docs/content/tutorial.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Getting Started - A Tutorial
+order: 7
+---
+
+<a name="tutorial"></a>
+
+# Getting Started with Spectacle: A Tutorial
+
+In this guide, we'll show you a couple of different ways to get started with Spectacle and walk you through the creation and customization of a presentation deck.
+
+<a name="option-one"></a>
+
+## Option One: Using a standard React-based web app
+
+1. Spin up a new React project using [`create-react-app`](https://github.com/facebook/create-react-app):
+
+    ```sh
+    npx create-react-app spectacle-tutorial
+    ```
+
+2. Install Spectacle by running `yarn add spectacle` or `npm i spectacle`.
+
+3. In `App.js`, replace the boilerplate content with this Spectacle starter:
+
+    ```jsx
+    import React from 'react';
+    import { Deck, Slide, Heading } from 'spectacle';
+
+    function App() {
+      return (
+        <Deck>
+          <Slide>
+            <Heading>Welcome to Spectacle</Heading>
+          </Slide>
+        </Deck>
+      );
+    }
+
+    export default App;
+    ```
+
+4. And you're good to go! Using `create-react-app`'s built-in `start` script, you can start a hot-reloading server to begin building your Spectacle presentation by running `yarn run start` or `npm run start`.
+
+<a name="option-two"></a>
+
+## Option Two: Using Markdown and the Spectacle CLI
+
+1. Create a new markdown file. You can use `.md` or `.mdx` (_MDX lets you mix JSX components inside markdown).
+
+    You can use this as a starter:
+
+      ```md
+      # Welcome to Spectacle
+
+      - This is a list item
+      - This is another list item
+
+      ---
+
+      # Second Slide
+
+      Text can be **bold** or _italic_!
+
+      Notes: These are presenter notes, only visible in presenter mode to the speaker.
+      ```
+
+    **Note:** The triple dash (`---`) is used as a slide delimiter. The `Notes:` keyword is used to embed presenter notes only visible to the speaker in presenter mode.
+
+2. To view your slides, supply your markdown to the Spectacle CLI to start a local web server.
+
+    `npx spectacle -s my-slides.mdx`
+
+3. And you're good to go! The web server you started supports live refreshing and will update your deck as you make changes to the markdown file.
+
+<a name="option-three"></a>
+
+## Option Three: Using One Page
+
+One Page is a single self-contained `HTML` file that lets you build a deck using no build steps, using [htm](https://github.com/developit/htm) over JSX to reduce the dependencies and load time.
+
+As a self-contained entity, it already has references to the dependencies you need to author and launch a deck in a web browser. Since there is no tooling required, One Page is also optimal on tablets. The One Page `HTML` file can be downloaded from the `examples` directory [in this repository](../../examples/one-page.html).
```

Comment:
GLOBAL: When we deploy, our root is `docs/`. Nothing below `docs/` is available, so we can't have any `../../` relative links. We need to:

1. Switch this to a github repo link or unpkg link (one-page is published). Just assume it's `master` even if not presently.
2. Scour everything in `docs/` to make sure that we haven't missed anything else like this.